Directory.Build.props

```diff
@@ -312,7 +312,7 @@
   <PropertyGroup>
     <!-- default to allowing all language features -->
     <LangVersion>latest</LangVersion>
-    <LangVersion Condition="'$(Language)' == 'C#'">8.0</LangVersion>
+    <LangVersion Condition="'$(Language)' == 'C#'">preview</LangVersion>
```

Comment:
For those interested in the new `LangVersion`, see https://devblogs.microsoft.com/dotnet/an-update-to-c-versions-and-c-tooling/